src/controllers/keystore/keystore.ts

```diff
@@ -439,9 +446,61 @@ export class KeystoreController extends EventEmitter {
 
     // Set up the cipher
     const counter = new aes.Counter(this.#mainKey!.iv) // TS compiler fails to detect we check for null above
-    const aesCtr = new aes.ModeOfOperation.ctr(this.#mainKey!.key, counter) // TS compiler fails to detect we check for null above
+    const aesCtr = new aes.ModeOfOperation.ctr(this.#mainKey!.key, counter) // TS compiler fails to detect we check for null above\
+    return hexlify(aesCtr.encrypt(new TextEncoder().encode(seed)))
+  }
+
+  async addSeedToTemp({ seed, hdPathTemplate }: KeystoreSeed) {
+    this.#tempSeed = {
+      seed: await this.#getEncryptedSeed(seed),
+      hdPathTemplate
+    }
+
+    this.emitUpdate()
+  }
+
+  async deleteTempSeed() {
+    this.#tempSeed = undefined
+    this.emitUpdate()
+  }
+
+  async moveTempSeed() {
```

Comment:
Minor, but "move" sounds bizarre. I'd name this in a more explicit manner like `addTempSeedToKeystoreSeeds`. But feel free to leave it as it, if that sounds better to you.